pip_review/__main__.py

```diff
@@ -118,7 +120,7 @@ def get_pkg_info(pkg_name, silent=False):
     return info
 
 
-def latest_version(pkg_name, prerelease=False, silent=False):
+def latest_version(pkg_name, pre_release=False, silent=False):
```

Comment:
In that case, please undo. We shouldn't rename things unless it solves a problem...